packages/core/tests/unit/models/apex-pie-test.js

```diff
@@ -0,0 +1,202 @@
+import { set } from '@ember/object';
+import { run } from '@ember/runloop';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { buildApexTestRequest } from '../../helpers/request';
+
+module('Unit | Model | Apex Pie', function(hooks) {
+  setupTest(hooks);
+
+  test('rebuildConfig formats correctly', function(assert) {
+    assert.expect(4);
+    const rows = [
+      {
+        m1: 1,
+        'd1|id': '1',
+        'd1|desc': 'Foo1',
+        'd2|id': '1',
+        'd2|desc': 'Bar1'
+      },
+      {
+        m1: 2,
+        'd1|id': '2',
+        'd1|desc': 'Foo2',
+        'd2|id': '2',
+        'd2|desc': 'Bar2'
+      },
+      {
+        m1: 3,
+        'd1|id': '3',
+        'd1|desc': 'Foo3',
+        'd2|id': '3',
+        'd2|desc': 'Bar3'
+      }
+    ];
+    const chart = run(() =>
+      run(() => this.owner.lookup('service:store').createRecord('all-the-fragments')).get('apexPie')
+    );
+    // testing one metric with one dimension
```

Comment:
Can we move this comment to be a string as the third parameter of the assert below?